src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonerResource.kt

```diff
@@ -141,6 +142,7 @@ class PrisonerResource(private val globalSearchService: GlobalSearchService) {
     ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]),
   )
   @Operation(summary = "List of offenders globally matching the offenderNo.", description = "List of offenders globally matching the offenderNo.")
+  @VerifyOffenderAccess(overrideRoles = ["SYSTEM_USER", "VIEW_PRISONER_DATA"])
```

Comment:
still unsure if need GLOBAL_SEARCH.  I initially dismissed this but a similar endpoint (/api/prisoners) endpoint allows multiple offenderNos which can be called instead - and this includes GLOBAL_SEARCH

Reply:
think it is fine not to include it